frontend/src/components/userprofile/EditProfile.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const EditProfile = () => {
+  return (
```

Comment:
Same thing as my comment on `profile.jsx`

The user will need a way to get back so consider a back button or using the Navbar that I created.